src/games/snake/snake_part.py

```diff
@@ -0,0 +1,24 @@
+import pygame
+from src.resources.snake_game_resources import SnakeGameResources
+
+
+class SnakePart(pygame.sprite.Sprite):
+    def __init__(self, belongs_to, board, pos_center = None):
+        pygame.sprite.Sprite.__init__(self, belongs_to)
+        self._part_size = board.get_field_size()
+        self.image = pygame.transform.scale(pygame.image.load(SnakeGameResources.SNAKE_PART_TEXTURE_PATH), self._part_size)
```

Comment:
Loading image is very time-consuming. You do it each time you create a new snake part. You can extract it to be a SnakePart static variable so it can be loaded only once. Later on, you can use it as a reference.